templates/workloads/services/lb-web/cf.yml

```diff
@@ -163,6 +163,14 @@ Resources:
               Action:
                 - elasticloadbalancing:DescribeRules
               Resource: "*"
+        - PolicyName: "ECSAccess"
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Effect: Allow
+              Action:
+                - ecs:ListTasks
+              Resource: "*"
```

Comment:
Can we restrict this to the copilot cluster? like in:
https://github.com/aws/copilot-cli/blob/aeb2e12dc98fa88a04c6ca704ea21df8d83afcf5/templates/workloads/scheduled-job/state-machine.yml#L53-L60